config.js

```diff
@@ -16,12 +16,20 @@ const ConfigOptions = () => {
     let config = {
         isProduction,
         isDev,
-
         webRootDir: '.',
         assetSrcDir: 'src',
         assetDistDir: 'dist',
         applyRevision: true,
 
+        /**
+         * Fozzie-related variables
+         */
+        fozzieSettings: {
+            useFozziePackageVersion: false,
+            getFozziePackageVersion: consumingPackageConfig && consumingPackageConfig.version,
+            fozzieBaseName: 'fozzie-'
```

Comment:
Not sure you'll need this – I'd just use the name of the SCSS file being generated unless you think there's a good reason for allowing customisation in the settings too?

Reply:
One of the requirements on the ticket was to version it with the fozzie name e.g. `fozzie-[version].css`, mostly why I put everything inside a fozzie object :)

Reply:
Ah I see! 👍